src/main/java/com/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,167 @@
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("It is not possible to update the order " + order, e);
+        }
+        return productInsert(order);
+    }
+
+    @Override
+    public boolean deleteById(Long orderId) {
+        String query = "UPDATE orders SET deleted = true WHERE order_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, orderId);
+            return statement.executeUpdate() == 1;
+        } catch (SQLException e) {
+            throw new DataProcessingException("It is not possible to delete the order with ID - "
+                    + orderId, e);
+        }
+    }
+
+    @Override
+    public List<Order> getAll() {
+        List<Order> orders = new ArrayList<>();
+        String query = "SELECT * FROM orders WHERE deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                orders.add(getOrderFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Unable to receive orders", e);
+        }
+        for (Order order : orders) {
+            order.setProducts(getProductFromOrder(order.getId()));
+        }
+        return orders;
+    }
+
+    private Order getOrderFromResultSet(ResultSet resultSet) throws SQLException {
+        Long orderId = resultSet.getLong("order_id");
+        Long userId = resultSet.getLong("user_id");
+        return new Order(orderId, userId);
+    }
+
+    private List<Product> getProductFromOrder(Long orderId) {
+        String query = "SELECT * FROM products "
+                + "INNER JOIN orders_products ON products.product_id = orders_products.product_id "
+                + "WHERE orders_products.order_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, orderId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Product> products = new ArrayList<>();
+            while (resultSet.next()) {
+                Long productId = resultSet.getLong("product_id");
+                String name = resultSet.getString("name");
+                BigDecimal price = resultSet.getBigDecimal("price");
+                products.add(new Product(productId, name, price));
+            }
+            return products;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get order with ID = " + orderId, e);
+        }
+    }
+
+    private Order productInsert(Order order) {
```

Comment:
remember about method naming (it should be a verb)

Reply:
Done.